README.md

```diff
@@ -60,9 +60,10 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- <p align="right">(<a href="https://mellow-cassata-728b9c.netlify.app/">Click here</a>)</p>
+- To see live Demo  <a href="https://mellow-cassata-728b9c.netlify.app/">Click here</a></p>
```

Comment:
- According to the project requirement, please deploy your website using **GitHub Pages** and update the README of your repository to include a link to the online version. 👍 😃 

![image](https://github.com/DamianDib9/personal-portfolio/assets/78251927/9c8d8861-39e2-4368-a38b-b71362470d92)

- Changes made to a GitHub Pages page do not appear as they are made, as GitHub uses CDN caching for Pages, and there is a small delay for changes to be reflected on the actual page. 😊

Reply:
OK, done 👍